scripts/create_singularities

```diff
@@ -453,6 +454,46 @@ def main(push: bool, githubids: list[str]|None) -> None:
         builder.runcmd("datalad", "push", "--data=auto")  # so we share with the world
 
 
+def retry_get(url: str) -> requests.Response:
+    sleepiter = exp_wait(attempts=10)
+    while True:
+        try:
+            return requests.get(url)
+        except (requests.ConnectionError, requests.Timeout) as e:
+            if (wait := next(sleepiter, None)) is not None:
```

Comment:
note 2 myself: first use of `:=` in the script, but was added in 3.8 and this script for internal use so should be totally fine